scripts/skills/actives/rf_gouge_skill.nut

```diff
@@ -0,0 +1,114 @@
+		this.m.IconDisabled = "skills/rf_gouge_skill_sw.png";
+		this.m.Overlay = "rf_gouge_skill";
+		this.m.SoundOnUse = [
+			"sounds/combat/strike_01.wav",
+			"sounds/combat/strike_02.wav",
+			"sounds/combat/strike_03.wav"
+		];
+		this.m.SoundOnHit = [
+			"sounds/combat/strike_hit_01.wav",
+			"sounds/combat/strike_hit_02.wav",
+			"sounds/combat/strike_hit_03.wav"
+		];
+		this.m.Type = ::Const.SkillType.Active;
+		this.m.Order = ::Const.SkillOrder.OffensiveTargeted;
+		this.m.IsSerialized = false;
+		this.m.IsActive = true;
+		this.m.IsTargeted = true;
+		this.m.IsStacking = false;
+		this.m.IsAttack = true;
+		this.m.IsIgnoredAsAOO = true;
+		this.m.IsTooCloseShown = true;
+		this.m.IsWeaponSkill = true;
+		this.m.InjuriesOnBody = ::Const.Injury.CuttingBody;
+		this.m.InjuriesOnHead = ::Const.Injury.CuttingHead;
+		this.m.DirectDamageMult = 0.3;
+		this.m.ActionPointCost = 6;
+		this.m.FatigueCost = 30;
+		this.m.MinRange = 1;
+		this.m.MaxRange = 2;
+		this.m.ChanceDecapitate = 75;
+		this.m.ChanceDisembowel = 50;
+		this.m.ChanceSmash = 0;
+	}
+
+	function getTooltip()
+	{
+		local ret = this.getDefaultTooltip();
+		ret.push({
+			id = 6,
+			type = "text",
+			icon = "ui/icons/hitchance.png",
+			text = ::Reforged.Mod.Tooltips.parseString("Has a " + ::MSU.Text.colorRed(this.getContainer().getActor().getCurrentProperties().IsSpecializedInCleavers ? "50%" : "33%") + " lower [threshold|Concept.InjuryThreshold] to inflict [injuries|Concept.InjuryTemporary]")
+		});
+
+		if (this.m.BleedStacks != 0)
+		{
+			ret.push({
+				id = 7,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Applies " + ::MSU.Text.colorGreen(this.m.BleedStacks) + " stacks of [Bleeding|Skill+bleeding_effect] when inflicting at least " + ::MSU.Text.colorRed(::Const.Combat.MinDamageToApplyBleeding) + " damage")
+			});
+		}
+
+		return ret;
+	}
+
+	function addResources()
+	{
+		foreach (r in this.m.SoundsA)
+		{
+			::Tactical.addResource(r);
+		}
+	}
+
+	function onAfterUpdate( _properties )
+	{
+		this.m.FatigueCostMult = _properties.IsSpecializedInCleavers ? ::Const.Combat.WeaponSpecFatigueMult : 1.0;
+	}
+
+	function onUse( _user, _targetTile )
+	{
+		this.spawnAttackEffect(_targetTile, ::Const.Tactical.AttackEffectChop);
+		return this.attackEntity(_user, _targetTile.getEntity());
+	}
+
+	function onBeforeTargetHit( _skill, _targetEntity, _hitInfo )
+	{
+		if (_skill == this)
+		{
+			_hitInfo.InjuryThresholdMult *= this.getContainer().getActor().getCurrentProperties().IsSpecializedInCleavers ? 0.5 : 0.66;
+
+			if (!_targetEntity.getCurrentProperties().IsImmuneToBleeding)
+				::Sound.play(this.m.SoundsA[::Math.rand(0, this.m.SoundsA.len() - 1)], ::Const.Sound.Volume.Skill, this.getContainer().getActor().getPos());
+		}
+	}
+
+	function onTargetHit( _skill, _targetEntity, _bodyPart, _damageInflictedHitpoints, _damageInflictedArmor )
+	{
+		if (_skill == this && _targetEntity.isAlive() && this.m.BleedStacks != 0)
```

Comment:
you do not check 
- how much damage was actually dealt and compare that to `::Const.Combat.MinDamageToApplyBleeding`. As of right now this will always apply bleeds on hit, even if it doesnt penetrate armor.
- whether the target is immune to bleeding